### ISSUE TITLE
Error adding Part Group

### ISSUE THREAD

**Issue by neilt** [ledgersmb/LedgerSMB#7726]
Master as of 3295d5e1d6cee2f11ba1b38892c584e3adf3116f (Wed Nov 22 07:06:23 AM CST 2023)

When I tried to enter a new group I get the following error.

Steps:

1. Login to new database with having only run setup.pl
2. Navigate to Goods and Services -> Add Group
3. Type 'special' into the group name
4. click 'Save'

<img width="1392" alt="Screenshot 2023-11-22 at 7 00 55 AM" src="https://github.com/ledgersmb/LedgerSMB/assets/1809796/4e40b0c9-da2a-4179-8f3b-da7611afd3c1">

**Comment by neilt** [comment]
Because of the error above, I added some part groups using SQL so I could test how they work.

```sql
insert into partsgroup (partsgroup) Values ('displays')
insert into partsgroup (partsgroup) Values ('special')
insert into partsgroup (partsgroup) Values ('computers')
```

Then I tried to create a part (Goods and Services -> Add Part) and assign a part group, but there were some problems.

1. after selecting a part group from the drop down, then clicking on 'Update' the part group selection disappears.
2. after selecting a part group again, then clicking 'Save' the part group selection disappears.
3. However, when go back and search the part, it appears to have correctly saved the part group.
4. The documentation (Book 25.1.2, page 126) indicates that a part can be a member of multiple part groups.  That did not look possible examining the import format, the parts table definition, and the UI, so I removed 'however, a good can be assigned to more than one part group.'. see https://github.com/ehuelsmann/ledgersmb-book/pull/65
